internal/gps/pkgtree/reachmap.go

```diff
@@ -20,44 +20,37 @@ type ReachMap map[string]struct {
 	Internal, External []string
 }
 
-// FlattenAll flattens a reachmap into a sorted, deduplicated list of all the
-// external imports named by its contained packages.
-//
-// If stdlib is false, then stdlib imports are excluded from the result.
-func (rm ReachMap) FlattenAll(stdlib bool) []string {
```

Comment:
Well we could add back a (nilable) `filter func(string) bool` parameter to both `FlattenOmitStdLib` and `FlattenFn` either now or in the future.